### DISCUSSION TITLE
Welcome to terraformer Discussions!

### DISCUSSION THREAD

**Discussion by sergeylanzman** [GoogleCloudPlatform/terraformer#703] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jessieaquino** [comment]
I am using cross account sts assume role with a temporary access keys and ids,  I keep getting this error:
plugin.terraform-provider-aws_v3.37.0_x5: 2021/04/22 11:27:45 [DEBUG] Trying to get account information via sts:GetCallerIdentity: timestamp=2021-04-22T11:27:45.335-0500
2021-04-22T11:27:45.561-0500 [INFO]  plugin.terraform-provider-aws_v3.37.0_x5: 2021/04/22 11:27:45 [DEBUG] Trying to get account information via sts:GetCallerIdentity: timestamp=2021-04-22T11:27:45.561-0500
2021/04/22 11:27:46 aws importing... s3
2021/04/22 11:27:46 aws error initializing resources in service s3, err: operation error S3: ListBuckets, https response error StatusCode: 403,, api error AccessDenied: Access Denied

**Comment by cloud-learner67** [comment]
Newbie here. How do I know which version of Terraformer supports which version of Terraform. Is there some documentation on this?

**Reply by sergeylanzman** [reply]
0.14 for now, but it's still syntax, maybe need auto update state file (like this `terraform state replace-provider -- -/aws hashicorp/aws`)

**Comment by msirajud** [comment]
Dear guys, first all of thanks for the wonderful tool. We have managed to import the state file per resource group, able to import all network related resources except vnet peering. we don't see the vnet peering resource listed in the resources available to import in the documentation. Is vnet peering not available or is it kind of development or anything we can expect sooner. Please advise any alternate way to handle it just in case.
